src/index.js

```diff
@@ -878,6 +879,11 @@ export class TextField extends MaterialComponent {
 		if (cl) {
 			(field.attributes = field.attributes || {}).class = cl;
 		}
+
+		if (errorMessage) {
+			let classes = (field.attributes = field.attributes || {}).class || "";
+			(field.attributes = field.attributes || {}).class = classes + " is-invalid";
```

Comment:
can you switch this to [setClass()](https://github.com/developit/preact-mdl/commit/a7fbba7dfc1dc290e8e6fc9f1ade917dd5b00440#diff-1fdf421c05c1140f6d71444ea2b27638R26)?

```js
setClass(field.attributes, 'is-invalid', true);
```